clients/client/src/main/java/org/projectnessie/client/ResultStreamPaginator.java

```diff
@@ -40,7 +40,7 @@
      *
      * @param ref reference
      * @param pageSize page size
-     * @param token paging token
+     * @param token paging continuation token
```

Comment:
nit: in `AbstractParams` it is called `pageToken`... WDYT about renaming this param (and the method in `PaginatedResponse`) to use the same `pageToken` base name?.. just for the sake of clarity and consistency.

For me personally the bare word `token` sound more like something related authentication than to paging.

Reply:
yup - opened #2719